pkg/nsx/services/vpc/vpc.go

```diff
@@ -585,7 +588,7 @@ func (s *VPCService) CreateOrUpdateVPC(obj *v1alpha1.NetworkInfo) (*model.Vpc, *
 
```

Comment:
line 577: the we should also call NSXLBEnabled or getLBProvider to set AVI flag in VPC API